### PR TITLE
Adds the jester costume set to Geoff's sell list

### PR DESCRIPTION
**Pull request by Sir-Mora**
## About the PR 
adds the jester costume set to Geoff's sell list, which has the hat, mask, jumpsuit, and shoes.

## Why's this needed? 
as far as I can tell there is currently no way to get the full set of items anymore, and that is quite the shame to clowns who want to wear something that isn't a clown outfit.

### CODE REVIEW

**Comment by NightmareChamillian** [comment]
I'd love if, while you're in there, you also made it so the jester hat doesn't hide folks's hair!

**Comment by Sir-Mora** [comment]
> I'm gonna be Mean Dev and say the jester outfit is pretty... luxurious? Should be like 300 credits so clowns have to work a bit for it

your not bein a mean dev! >:P
I was already on the fence about that so I got no problem with making it a similar price to other costumes

File: code/obj/item/storage/clothing.dm [hunk 1]
```diff
@@ -292,6 +292,15 @@
 		/obj/item/clothing/shoes/black,
 	)
 
+/obj/item/storage/box/costume/jester
+	name = "jester costume"
+	desc = "a box that contains a jesters outfit"
```
Comment (flrsh) [inline code/obj/item/storage/clothing.dm:297]:
```suggestion
	desc = "A box that contains a jester's outfit"
```

File: code/modules/economy/commodity.dm [hunk 2]
```diff
@@ -2242,6 +2242,15 @@
 	comtype = /obj/item/storage/box/costume/mime/alt
 	desc = "This stuff will give you an edge in charades."
 
+/datum/commodity/costume/jester
+	comname = "Jester Costume Set."
+	comtype = /obj/item/storage/box/costume/jester
+	desc = "Travel back in time and become the medieval version of a clown. (does not provide time travel)"
```
Comment (flrsh) [inline code/modules/economy/commodity.dm:2248]:
```suggestion
	desc = "Travel back in time and become the medieval version of a clown. (Does not provide time travel)"
```

**Review by flrsh — APPROVED** [review]

**Pull request merged**